lib/internal/cli_table.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const ansiRegex = () =>
+  new RegExp([
+    '[\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:[a-zA-Z\\d]*',
+    '(?:;[a-zA-Z\\d]*)*)?\\u0007)',
+    '(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PRZcf-ntqry=><~]))',
+  ].join('|'), 'g');
```

Comment:
Without looking more into it: please add a description what this regular expression is for? And I am not sure why it is created again on each call? It should only be necessary to create the RegExp once?